wifiphisher.py

```diff
@@ -775,6 +797,7 @@ def get_hostapd():
     # Main loop.
     try:
         while 1:
+            time.sleep(5)
```

Comment:
The rapid refresh rate of the os.system("clear") causes periodic terminal hangs which disrupts the output therefore making the main status screen unusable (Kali Linux)